services-directions-models/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     // AutoValue
     compileOnly dependenciesList.autoValue
     compileOnly dependenciesList.autoValueGson
+    annotationProcessor(dependenciesList.autoValueGson)
```

Comment:
Why do we need this now?

Reply:
The gradle wrapper needed to be updated to support to the sdk-registry minimum version. The new autoValueGson in the new gradle versions were not being generated

[here was an error](https://app.circleci.com/pipelines/github/mapbox/mapbox-java/309/workflows/f38e942b-17ca-4611-8eb1-f0c61504e668/jobs/4672)
```
...
> Task :services:compileJava FAILED
/root/code/services-directions/src/main/java/com/******/api/directions/v5/MapboxDirections.java:353: error: package AutoValue_MapboxDirections does not exist
    return new AutoValue_MapboxDirections.Builder()
                                         ^
1 error

FAILURE: Build failed with an exception.
```

It was confusing, but got the idea from the autovalue gson page and it fixed it
https://github.com/rharter/auto-value-gson#download

Reply:
gotcha